lms/static/certificates/sass/_print.scss

```diff
@@ -94,6 +94,12 @@
 // ------------------------------
 @media print {
 
+    html,
+    body {
+        height: 650px;
```

Comment:
Also, I'm not sure what's the lesser of two evils...

* a cert properly rendering + an extra piece of paper/page in a doc 
* the chance that this height restriction may cause cert content to be cut off or not properly displayed

@frrrances, do you have any quick thoughts on what the better option is?